### DISCUSSION TITLE
Multiple isolated fungus_mt components detected! Broken or contamination?

### DISCUSSION THREAD

**Discussion by phnghia99** [Kinggerm/GetOrganelle#229] [Q&A]
My fungus_mt has Disentangling failed by multiple fungus_mt components. This is my log file:
[get_org.log.txt](https://github.com/Kinggerm/GetOrganelle/files/10372078/get_org.log.txt)

**Comment by Kinggerm** [comment]
https://github.com/Kinggerm/GetOrganelle/wiki/FAQ#what-should-i-do-with-incomplete-resultbroken-assembly-graph
